forwards_paper.tex

```diff
@@ -554,10 +554,49 @@ \subsection*{Tree sequence simplification}
 (These lineages are the ``colors'' above.)
 We also maintain a map $A$ such that $A_j$ is segment chain for node $j$ in
 the input tree sequence.
-Crucially, the time required to run the algorithm is linear in the number of edges of the input tree sequence.
+
+\begin{figure}
+    \begin{center}
+        \includegraphics[width=10cm]{simplify-benchmarks}
+    \end{center}
+    \caption{Scaling of the simplify algorithm in terms of the number of
+        input edges (A) and size of the sample (B). In (A) we recorded the
+        entire set of edges in a forwards-time simulation with $N=1000$
+        and $\rho = 10000$ for $20N$ generations, giving around
```

Comment:
I vote for reporting the sequence length (in cM) here, with rho as a parenthetical remark.

Reply:
The length in cM will be a wacky number.  Little r per diploid is 1e4/4e3 = 2.5.  The distance in cM is the probability that a Poisson deviate with mean 2.5 is odd, or exp(-2.5)*sinh(2.5) IIRC, which is ~0.49, or roughly 50cM.

Reply:
I'm agnostic on this, it's up to you two.

Reply:
Oh, right. I have a bad habit of thinking of Morgans as units of "expected number of crossovers", which would be 2.5 here, no?  Kinda like human chr1?  So, how about
```
with $N=1000$ and $\rho=10000$ for $20N$ generations
(roughly the length of human chromosome 1).
```

Reply:
> Oh, right. I have a bad habit of thinking of Morgans as units of "expected number of crossovers", which would be 2.5 here, no? Kinda like human chr1? So, how about

Yeah, it is a tricky thing. In a very literal sense, 50cM is about 2/3 the length of chr21/chr22.  In the sense of expected number of crossovers, we're closer to something in the chr1-chr3 range.

That said, I do like the fig as-is.